src/content/tools/devtools/deep-links.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Validate deep links
+description: Learn how to validate deep links in your app.
+---
+
+:::note
+The deep linking validator was added to DevTools
+in Flutter SDK 19. It currently works only for
+Android, but will be expanded to cover iOS and
+web in future releases.
+:::
+
+The deep linking view validates the deep links
+on your app.
+
+You can open DevTools, click into the **Deep Links** tab,
+and import a Flutter project that contains deeplinks.
```

Comment:
Do we need a screen for import page and a screen of the validating page?

Reply:
@chunhtai, whatever you think is best. If two screenshots are better than one, let's use two.  🗡️ 

And, thanks for the review!

Reply:
I'm going to land with one screenshot, @chunhtai. But, in future, it would be nice to improve this page a bit when we have more time.